recirq/otoc/otoc_example.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Example for measuring OTOC of several random circuits."""
+
+import cirq
+import numpy as np
+from matplotlib import pyplot as plt
+
+from recirq.otoc.otoc_circuits import build_otoc_circuits
+
+
+def main():
+    # Specify project ID and processor name.
+    project_id = ''
+    engine = cirq.google.Engine(project_id=project_id,
+                                proto_version=cirq.google.ProtoVersion.V2)
+    processor_name = 'rainbow'
```

Comment:
Use `cirq.google.get_engine_sampler` which will use the GOOGLE_CLOUD_PROJECT environment variable, if not otherwise provided. This will also return a `QuantumEngineSampler`, which is how we want users to be executing programs since it can be substituted for a simulator

Reply:
Done